bbs+.js

```diff
@@ -0,0 +1,22 @@
+const { Bls12381G2KeyPair } = require("@transmute/did-key-bls12381");
+const { BbsBlsSignature2020 } = require("@mattrglobal/jsonld-signatures-bbs");
+
+const ISSUER_KEY_PAIR_SEED_BASE64 = "2owx9/ylDJazW2oubRDPZSZGKzhSRPud1dbJGkufavk=";
+
+async function main () {
+
+    const keypair = await Bls12381G2KeyPair.generate({
+        secureRandom: () => {
+            return Buffer.from(ISSUER_KEY_PAIR_SEED_BASE64,'base64');
+        },
+    });
+    keypair.id = keypair.controller + keypair.id;
+    const suite = new BbsBlsSignature2020({
+        key: keypair,
+    });
+
+    console.log (keypair);
```

Comment:
Unsure what this function is doing? Just logging the generated key pair?

Reply:
@tplooker ... forgot to remove this file. was testing out stuff on bbs+ and forgot to remove after I was done